src/style.css

```diff
@@ -1,3 +1,78 @@
 body {
   margin: 0;
+  width: 100%;
+
+  font-size: 12px;
+  text-transform: uppercase;
+
+  --bluelink: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  background-color: white;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  display: flex;
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+
+  list-style: none;
+}
+
+.nav__link {
+  display: inline-block;
+  margin-right: 20px;
+  height: 60px;
+
+  line-height: 60px;
+  color: black;
+  text-decoration: none;
+}
+
+.nav__link-last {
+  margin-right: 0;
+}
+
+.nav__link--blue {
```

Comment:
you don't need this class `nav__link--blue`, use `is-active` for reach this goal